app/views/answer_mailer/notify_question_author.html.erb

```diff
@@ -1,23 +1,18 @@
-Hi! A new answer has been posted for your question '<a href='https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>'><%= @answer.node.title %></a>'. Do NOT reply to this email; click this link to see the answer:
+Hi! <a href='https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @answer.author.name %>'><%= @answer.author.name %></a> responded :
+<p><%= @answer.content.truncate(30) %></p>
 
-<p>https://<%= ActionMailer::Base.default_url_options[:host] %><%= @answer.node.path(:question) %>#a<%= @answer.id %></p>
+Do NOT reply to this email; click this link to see the answer:
 
-<p><b>Here is the answer that <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/profile/<%= @answer.author.name %>"><%= @answer.author.name %></a> wrote:</b></p>
-
-<hr /> 
-
-<p><%= raw auto_link(RDiscount.new(@answer.body_email(ActionMailer::Base.default_url_options[:host])).to_html) %></p>
```

Comment:
I think we're not quite ready to remove the full text from the email -- maybe we could revert this and think that through as a follow-up? I'd like to keep the entire text in for now. I'm not as worried about spammers, and think it's really great for a "quick look" at a new post. Thanks!